ui/apps/platform/src/services/DeploymentsService.ts

```diff
@@ -92,7 +120,7 @@ export function fetchDeploymentsLegacy(
     }
     const params = queryString.stringify(queryObject, { arrayFormat: 'repeat', allowDots: true });
     return axios
-        .get<{ deployments: ListDeploymentWithProcessInfo[] }>(`${deploymentsUrl}?${params}`)
+        .get<{ deployments: ListDeploymentWithProcessInfo[] }>(`${deploymentsWithProcessUrl}?${params}`)
```

Comment:
It looks like style checks reports a problem here with need for wrapping because of line length.